spec/config/initializers/rack_attack_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "securerandom"
+
+describe Rack::Attack, type: :request do
+  subject(:post_request) do
+    post user_session_path, params: params, headers: headers, as: :json
+  end
+
+  let(:limit) { 5 }
+
+  describe 'throttle excessive requests by email address' do
+    let(:params) do
+      {
+        user: {
+          email: 'wrong@email.com',
+          password: 'wrong_password'
+        }
+      }
+    end
+
+    let(:headers) { {} }
+
+    context 'number of requests is lower than the limit' do
```

Comment:
```suggestion
    context 'when the number of requests is lower than the limit' do
```

Reply:
Same comment for the other `context` blocks
https://rspec.rubystyle.guide/#context-descriptions